### ISSUE TITLE
Exception during MQTT homeassistant export for production PDL only after 0.8.13

### ISSUE THREAD

**Issue by ioull** [MyElectricalData/myelectricaldata#343]
I have a PDL with only production, no consumption (old contract with edf-oa)
It was working well in 0.8.13 but since I update it in 0.8.15 then 0.8.16 I have the following exception :
![image](https://user-images.githubusercontent.com/6410144/232693249-0c8027ca-3fab-4113-a89e-e24707571ba5.png)

Here is my config : 
![image](https://user-images.githubusercontent.com/6410144/232693117-d1ae1ddd-471a-4c24-a47d-aeb973842129.png)                                                                                                                                                                                 

Sorry for the image ... I'm not able to copy/paste my logs without ugly presentation ...

**Comment by m4dm4rtig4n** [comment]
Merci de respecter le template de création des issues.

**Comment by ioull** [comment]
Désolé, les templates m'avaient pas été proposés quand j'ai cliqué sur 'new issue' ... Je recommence ...
